pages/docs/manual/latest/attribute.mdx

```diff
@@ -45,6 +45,12 @@ type student = {
   age: int,
   @bs.as("aria-label") ariaLabel: string,
 }
+
+@deprecated
+let customDouble = foo => foo * 2
+
+@deprecated("Do the math yourself!")
```

Comment:
```suggestion
@deprecated("Use SomeOther.customTriple instead")
```
Would probably be a more practical use-case

Reply:
Agreed, makes sense.

Reply:
I needed to also update the explanation. After this builds, I believe it is ready.